.github/workflows/quay.yml

```diff
@@ -0,0 +1,16 @@
+name: quay
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    steps:
+    - name: update container on quay.io
+      run: curl -H $header --data $payload ${{ secrets.QUAY_WEBHOOK_URL }}
```

Comment:
You should add `run: set -x; curl ...` so it leaks the `${{ secrets.QUAY_WEBHOOK_URL }}` in the VM.

Reply:
Don't we not want the secret in the VM?

Reply:
Sure, but I was just messing around with that comment. All `set -x` is doing is increased verbosity of the logs, and should, in theory, leak the value of the `{{ secrets.QUAY_WEBOOK_URL }}` unless there's logic in place in the actions that would prevent that.